mutter: Fix broken includes for xcursor

  Add missing include for the xcursor header to required
  files.

  Upstream-status: Pending

  Signed-off-by: Brendan Kerrigan <kerriganb@ainfosec.com>

diff --git a/src/core/display.c b/src/core/display.c
index f6b4d5b9f..3930f1c32 100644
--- a/src/core/display.c
+++ b/src/core/display.c
@@ -54,6 +54,7 @@
 #include <meta/meta-backend.h>
 #include "backends/meta-cursor-sprite-xcursor.h"
 #include "backends/meta-logical-monitor.h"
+#include "backends/meta-cursor-sprite-xcursor.h"
 #include "backends/native/meta-backend-native.h"
 #include "backends/x11/meta-backend-x11.h"
 #include "backends/meta-stage-private.h"
diff --git a/src/x11/meta-x11-display.c b/src/x11/meta-x11-display.c
index 83f7dfbae..36ea95b91 100644
--- a/src/x11/meta-x11-display.c
+++ b/src/x11/meta-x11-display.c
@@ -55,6 +55,8 @@
 #include "backends/meta-logical-monitor.h"
 #include "backends/meta-settings-private.h"
 #include "backends/x11/meta-backend-x11.h"
+#include "backends/meta-cursor-sprite-xcursor.h"
+
 #include "core/frame.h"
 #include "core/meta-workspace-manager-private.h"
 #include "core/util-private.h"
-- 
2.17.1

